tembo-cli/src/cmd/top.rs

```diff
@@ -34,7 +34,7 @@ pub struct MetricsResponse {
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct MetricsData {
-    pub resultType: String,
```

Comment:
I think this change effects parsing, so please check still works and maybe can add a test for tembo top in the test against cloud. To fix the lint issue, it’s possible to use serde macros to parse the value as something different than the variable name.

Reply:
Oh yes.. I'm gonna revert the change.